README.md

```diff
@@ -43,20 +41,11 @@ The remainder of the repository contents consist of:
 - [docs](/docs) - The Hanko documentation
 
 # Getting started
+1. Try our hosted [live example](https://example.hanko.io) and our companion page [passkeys.io](https://passkeys.io) or use the [quickstart](/quickstart/README.md) to get a feel for the user experience provided by an application that leverages the Hanko backend API and our custom web component
+2. Head over to the [backend](/backend/README.md) to learn how to get it up and running for your own project
```

Comment:
Maybe add hint to Hanko Cloud? I.e.: "Launch your project on Hanko Cloud or host the backend yourself"?

Reply:
done